armi/bookkeeping/report/reportingUtils.py

```diff
@@ -180,6 +182,11 @@ def _writeMachineInformation():
                 nodeMappingData.append(
                     (uniqueName, numProcessors, ", ".join(matchingProcs))
                 )
+                # Run sys info on each unique node too
```

Comment:
Can we change this comment to something like:

```python
# If this is on Windows: run sys info on each unique node too
```

Reply:
done!